scheduler/planner.go

```diff
@@ -258,7 +258,7 @@ func (u *unitInfo) value() int64 {
 	// Increase the value for the number of dependents, so that
 	// tasks (and units) which block other tasks run before tasks
 	// that don't block other tasks.
-	value += priority * (u.NumDependents / length)
+	value += priority * (u.Settings.GetNumDependentsFactor() / 10) * (u.NumDependents / length)
```

Comment:
If we want it to be multiplicative factor, could we just make it a float instead of an int? That way we can reduce the factor as much as we wanted (i.e. `0.1` would be equivalent).